app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+    <%= t("signup.error") %>
+    <%= pluralize(@user.errors.count, "error") %>.
```

Comment:
1. `"error"` sử dụng I18n em nhé
2. Phân biệt size, count và length nào

Reply:
Phân biệt size, count và length.
-count: Nó sẽ dùng câu lệnh sql (SELECT COUNT() FROM…) để truy vấn ra số lượng của activerecord mà chúng ta truy vấn.
-length: Nó sẽ đếm số lượng mà không sử dụng thêm câu truy vấn nếu đối tượng đó đã được nạp, ngược lại, nó sẽ nạp toàn bộ object vào bộ nhớ và thực hiện đếm.
-size: Khi object đã đuợc load, nó sẽ thực hiện trên chính object đó trong bộ nhớ (không query vào database, tuơng tự lenght ở lần thứ 2 trở đi), còn khi object đó chưa được load, nó sẽ thực hiện câu lệnh (SELECT COUNT(*) FROM…) (tương tự như count)